admin_pages/general_settings/General_Settings_Admin_Page.core.php

```diff
@@ -557,13 +576,24 @@ protected function _update_admin_option_settings()
 
 
     /**
+     * @param string|null $default
      * @return string
      */
-    protected function getCountryIsoForSite()
+    protected function getCountryISO(?string $default = null): string
     {
-        return ! empty(EE_Registry::instance()->CFG->organization->CNT_ISO)
-            ? EE_Registry::instance()->CFG->organization->CNT_ISO
-            : 'US';
+        $default = $default ?? $this->getCountryIsoForSite();
+        $CNT_ISO = $this->request->getRequestParam('country', $default);
+        return strtoupper($CNT_ISO);
+    }
+
+
+    /**
+     * @return string
+     */
+    protected function getCountryIsoForSite(): string
+    {
+        return ! empty(EE_Registry::instance()->CFG->organization->CNT_ISO) ? EE_Registry::instance(
+        )->CFG->organization->CNT_ISO : 'US';
```

Comment:
```suggestion
        return ! empty(EE_Registry::instance()->CFG->organization->CNT_ISO)
            ? EE_Registry::instance()->CFG->organization->CNT_ISO
            : 'US';
```

Reply:
ug... stoopid autoformat 👊🏻 
missed that, thnx, will fix